azure-container-apps-arm/README.md

```diff
@@ -0,0 +1,103 @@
+   - **Workspace Credentials** *(Only if you are using Google Workspace as your identity provider)*: Paste the entire contents of the json key file you downloaded in the [before you begin](#before-you-begin) section. If you are not using Google Workspace as your identity provider, leave this field blank.
+> [!IMPORTANT]
+> If you are not using Google Workspace as your identity provider, leave `Workspace Actor` and `Workspace Credential` fields blank.
+3. Click **Review + create**.
+4. Once the validation succeeds, click **Create**. It will take a couple of minutes to run the deployment.
+
+
+## Step 3: Test your SCIM bridge
+Once your deployment is complete, click **Go to resource group** and click on the Container App you created.
+
+To test if your SCIM bridge is online, choose **Overview** in your application's sidebar, then click your **Application Url** link. This is your **SCIM bridge URL**. Sign in using your bearer token to verify that your SCIM bridge is connected to your 1Password account.
+
+## Step 4: Connect your identity provider
+
+To finish setting up automated user provisioning, [connect your identity provider to the SCIM bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider). 
+
+> [!IMPORTANT]
+> If you are using Google Workspace as your identity provider, you can skip this part. Once you sign in to your SCIM bridge using your bearer token as in step 3, you will see that your SCIM bridge is already connected to your Google Workspace account.
+
+
+<hr>
+
+## Update your SCIM bridge in the Azure Portal
+
+> [!TIP]
+> Check for 1Password SCIM Bridge updates on the [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
+
+1. Within your deployed 1Password SCIM Bridge Container App in the [Azure Container Apps Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.App%2FcontainerApps), select **Containers** from the sidebar.
+2. Click **Edit and deploy**.
+3. Select the checkbox next to your **op-scim-bridge** container, then choose **Edit**.
+4. Change the version number **2.9.5** in the **Image and Tag** field, **1password/scim:v2.9.5** to match the latest version from our [SCIM bridge releases notes website](https://releases.1password.com/provisioning/scim-bridge/).
+5. Select **Save**.
+6. Select **Create** to deploy a new revision using the updated image.
+7. Enter your SCIM bridge URL in a browser and sign in with your bearer token.
+8. Check the top left-hand side of the page to verify you're running the updated version of the SCIM bridge.
+
+After you sign in to your SCIM bridge, the [Automated User Provisioning page](https://start.1password.com/integrations/active/) in your 1Password account will also update with the latest access time and SCIM bridge version.
```

Comment:
We should maybe consider this in a new PR, but I feel like applying the template against the existing deployment should be the way we instruct folks to update this.